api/deps.sh

```diff
@@ -29,7 +29,20 @@ apt-get install -y \
     libssl-dev \
     pkg-config \
     libiberty-dev \
-    git-core
+    git-core \
+    wget
+
+echo "installing openssl"
+
+pushd $(/bin/pwd)
+cd /tmp && \
+wget https://www.openssl.org/source/openssl-1.1.0.tar.gz
```

Comment:
Any particular reason why we shouldn't use a git submodule (to https://github.com/openssl/openssl/commit/577d7010c6ffa98e8dc61d8826693db9d8f57bd5 or whatever) like the rest of the deps?